test/upgrade/test_main.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package upgrade
+
+import (
+	"os"
+	"testing"
+)
+
+// RunMainTest is a module TestMain.
+func RunMainTest(m *testing.M) {
+	os.Exit(m.Run())
+}
```

Comment:
This looks like does nothing. Maybe we can remove it?

Reply:
Done